### PR TITLE
Show "Public" indicator on the event page

### LINKED ISSUE #1819: Show "Public" indicator on public events/categories
This applies to the top right bar, where we currently only show "Restricted" events.
As for the conference/category list, I would leave it as it is, since it's easy to figure out from the context.
Logged out users should probably never see any of this, as they would assume the content is public.

### PR DESCRIPTION
**Pull request by mic4ael**
fixes #1819

### CODE REVIEW

File: indico/MaKaC/webinterface/tpls/ProtectionWidget.tpl [hunk 1]
```diff
@@ -3,6 +3,8 @@
     <div class="protectionWidgetSection">
         % if protection[0] == "DomainProtected":
             ${_("The information on this web page is restricted for display on the %s") % protection[1]}
+        % elif protection[0] == "Public":
+            ${_("The information on this web page is accessible and visible to anyone.")}
```
Comment (ThiefMaster) [inline indico/MaKaC/webinterface/tpls/ProtectionWidget.tpl]: I'd get rid of the *accessible and*.

**Comment by ThiefMaster** [comment]
> Logged out users should probably never see any of this, as they would assume the content is public.

You can check `_session.user` to see if the user is logged in.

**Comment by mic4ael** [comment]
But, isn't event/category inaccessible when its protection level is `!=` Public?

**Comment by mic4ael** [comment]
I mean, if event is public, then user that is not logged can see that event so we can indicate that this event is indeed Public.

**Comment by mic4ael** [comment]
Yeah, looks better when green.

**Comment by pferreir** [comment]
Ah, and I think the icon shouldn't be a shield, that's confusin. I suggest a globe (or something we may already have in the icon pack that looks as good).

**Comment by pferreir** [comment]
As the original ticket says, this shouldn't be visible is the user is not logged in, as everything will be logically public.

**Pull request merged**